### PR TITLE
Added RV Min Aggregators

### PR DESCRIPTION
**Pull request by jigold**

### CODE REVIEW

File: src/main/scala/is/hail/annotations/aggregators/RegionValueMinAggregator.scala [hunk 1]
```diff
@@ -0,0 +1,173 @@
+package is.hail.annotations.aggregators
+
+import is.hail.annotations._
+
+class RegionValueMinBooleanAggregator extends RegionValueAggregator {
+  private var min: Boolean = false
+  private var empty: Boolean = true
+
+  def seqOp(i: Boolean, missing: Boolean) {
+    if (!missing && (i < min || empty)) {
```
Comment (catoverdrive) [inline src/main/scala/is/hail/annotations/aggregators/RegionValueMinAggregator.scala]: would prefer `empty || i < min` for all of these, so the `i < min` check isn't redundant for the `empty` case.

Comment (jigold) [reply src/main/scala/is/hail/annotations/aggregators/RegionValueMinAggregator.scala]: I thought I was saving operations by doing it this way bc the common case is `empty` will be true.

Comment (jbloom22) [reply src/main/scala/is/hail/annotations/aggregators/RegionValueMinAggregator.scala]: @cseed any advice on `||` versus `|` on simple Boolean expressions like above?

Comment (catoverdrive) [reply src/main/scala/is/hail/annotations/aggregators/RegionValueMinAggregator.scala]: ok. that's probably true.

Comment (jigold) [reply src/main/scala/is/hail/annotations/aggregators/RegionValueMinAggregator.scala]: Meant `empty` will be false in most cases.

**Pull request merged**